nature_importer.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+import argparse
+import csv
+import os
+import importer_utils as utils
+import wikidataStuff.wdqsLookup as lookup
+from NatureArea import NatureArea
+from PreviewTable import PreviewTable
+from Uploader import Uploader
+
+DATA_DIRECTORY = "data"
+reserves_file = "NR_polygon.csv"
+nationalparks_file = "NP_polygon.csv"
+edit_summary = "test"
+
+
+def get_data_from_csv_file(filename):
+    """Load data from csv file into a list."""
+    with open(filename, "r") as f_obj:
+        reader = csv.DictReader(f_obj, delimiter=',')
+        csv_data = list(reader)
+    return csv_data
+
+
+def remove_duplicate_entries(nature_dataset):
+    """
+    Remove duplicate entries from Nature Reserve file.
+
+    In cases where a reserve occurs twice, it's because
+    of different values of "BESLSTATUS". Whenever that happens,
+    one of them is always "gällande". This is the one we keep,
```

Comment:
Are there cases where the status is not "gällande" but it is not a duplicate? If so do we also want to get rid of these?